src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/propertysettings/propertysettings.html

```diff
@@ -84,7 +83,7 @@
                                     </umb-node-preview>
                                     
                                     <div ng-if="vm.changeDataTypeHelpTextIsVisible" class="mt2 umb-alert umb-alert--info">
-                                        <localize key="contentTypeEditor_changeDataTypeHelpText">
+                                      <localize key="contentTypeEditor_changeDataTypeHelpText">Changing a data type with stored values is disabled. To allow this you can change the Umbraco:CMS:DataTypes:CanBeChanged setting in appsettings.json.</localize>
```

Comment:
Could you add this text also as label/translation in the en.xml lang file (located in Umbraco.Core - EmbeddedResources\lang)? This way it will be caught when people translate in other languages 😁
Thanks!

Reply:
Seems to be an unrelated item, I've reverted this part.